pandas/core/dtypes/inference.py

```diff
@@ -417,8 +418,6 @@ def is_dataclass(item):
 
     """
     try:
-        import dataclasses
```

Comment:
this is in a try/except for ImportError.  is the ImportError no longer possible?

Reply:
that's right, we're Python3.8+ now. @Thextan could you remove the `except` branch too please?

Reply:
can you address this please?